src/main/webapp/app/course/manage/exercises/create-modal/exercise-add-modal.component.scss

```
@@ -0,0 +1,168 @@
+        line-height: 1.4;
+        margin-bottom: 0.75rem;
+        flex: 1;
+    }
+}
+
+.card--programming {
+    --card-accent: #6366f1;
+}
+.card--quiz {
+    --card-accent: #f59e0b;
+}
```

Stylelint (17.13.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)